variables.tf

```diff
@@ -25,9 +25,29 @@ variable "public_secondary_zones" {
     delegation_set = string,
   }))
   default     = []
-  description = "Route53 secondary zone ('parent' zone must be specified as well). Set delegation_set to 'null' to use no delegation set. Use empty 'ns_servers' list to use AWS default nameserver."
+  description = "Public Route53 secondary zone ('parent' zone must be specified as well). Set delegation_set to 'null' to use no delegation set. Use empty 'ns_servers' list to use AWS default nameserver."
```

Comment:
By reading this, it's not exactly clear to me, what this "secondary zone" is...

Reply:
Also, IIRC, a secondary zone is something that can be promoted to master, so I'd say that this term is not used correctly here.